doc/source/skips/4-transition-to-v2.rst

```diff
@@ -143,29 +148,50 @@ namespace with a version number on it, such as OpenCV (imported as ``cv2``) and
 
-The details of the proposal are as follows:
-
-- scikit-image 0.19 will be followed by scikit-image 1.0. Every deprecation
-  message will be removed from 1.0, and the API will be considered the
-  scikit-image 1.0 API.
-- After 1.0, the main branch will be changed to (a) change the import name to
-  skimage2, (b) change the package name to skimage2, and (c) change the version
-  number to 2.0-dev.
-- There will be *no* "scikit-image" package on PyPI with version 2.0. Users who
-  ``pip install scikit-image`` will always get the 1.x version of the package.
-  To install scikit-image 2.0, users will need to ``pip install skimage2``,
-  ``conda install skimage2``, or similar.
-- After consensus has been reached on the new API, skimage2 will be released.
-- Following the release of skimage2, a release of scikit-image 1.1 is made. This
-  release is identical to 1.0 (including bugfixes) but will advise users to
-  either (a) upgrade to skimage2 or (b) pin the package to ``scikit-image<1.1``
-  to avoid the warning.
-- scikit-image 1.0.x and 1.1.x will receive critical bug fixes for an
-  unspecified period of time, depending on the severity of the bug and the
-  amount of effort involved.
+As a first execution step of this SKIP, scikit-image 1.0 will be released, celebrating the the maturity of the project.
+
+First phase: Building `skimage2`
+................................
+
+Afterwards, a new empty ``skimage2`` namespace will be created in our repository alongside our ``skimage`` namespace.
+It will be marked as experimental – importing it will warn that content in ``skimage2`` is still unstable.
+Initially, this namespace will not be included full releases (versioned 1.x) on PyPI and elsewhere, but may already be included in our nightly releases.
+Towards the end of this phase, ``skimage2`` should be included in full releases to help downstream libraries and users try it out.
+
+With the new namespace available, we will start building the new API inside it.
+It will – where possible – wrap the existing implementation in the existing ``skimage`` namespace but will have its own independent test suite.
+
+While ``skimage2`` will be a new API, we will try to keep the differences to the old API reasonably small to make the eventual transition easier for users.
+As a general rule it should always be possible to achieve the current behavior of the ``skimage`` API by some call or set of calls with the ``skimage2`` API.
+There may be some situations where we have to break this general rule, but an argument should be made for the relevant change that breaks this rule.
+
+We will record the pathway for migrating from the old to the new API in detail in a migration guide, in the docstrings of the old API and with the help of runtime warnings.
+
+During this phase, new (additive) features can still be introduced into the old ``skimage`` namespace, not only in the new one.
+If possible, their internal implementation should be added in the new namespace.
```

Comment:
One reason I'd like to see `skimage2` included in all releases early is, because it gives us the ability to add completely new implementations in the `skimage2` namespace from the start.

@stefanv, have you thought about this?